UI/Pages/MainPage.cs

```diff
@@ -66,5 +68,11 @@ public MainPage ClickToSideMenu()
             WebDriverExtension.ClickWithAction(openSideMenuButton);
             return new MainPage();
```

Comment:
Why functions has bool type but new MainPage() as returned? I see that ClickWithAction doing something on MainPage, but it's make some missunderstanding. You can change it:
1. Name function OpenSideMenu
OR
2. Do not change name, but put waiter in function instead of ClickWithAction

Reply:
But it is not returning a bool type, am I right? I have changed the function name

Reply:
I will ask it from its contributor, Mikita